layer/training/runtime/model_trainer.py

```diff
@@ -50,6 +126,72 @@ def __enter__(self) -> None:
     def get_working_directory(self) -> Path:
         pass
 
+    @staticmethod
+    def generate_system_metrics(
+        start_time: int, start_cpu_used: int, logger: Logger
+    ) -> Tuple[Dict[str, float], int]:
+        global cpu_used_temp  # pylint: disable=invalid-name
+        global start_time_temp  # pylint: disable=invalid-name
+
+        def _read_value_from_file(path: str) -> int:
+            with open(path, "r") as f:
+                return int(f.read())
+
+        def _get_mem_used() -> int:
+            return _read_value_from_file("/sys/fs/cgroup/memory/memory.usage_in_bytes")
+
+        def _get_mem_available() -> int:
+            return _read_value_from_file("/sys/fs/cgroup/memory/memory.limit_in_bytes")
+
+        def _get_used_percent(used: float, available: float) -> float:
+            if not used or not available:
+                print("System metric 0")
```

Comment:
Is this intentional?

Reply:
Removed the print and updated the logic slightly